subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/StatsDKubernetesReservationListener.kt

```diff
@@ -0,0 +1,42 @@
+package com.avito.android.runner.devices.internal.kubernetes
+
+import com.avito.android.runner.devices.model.ReservationData
+import com.avito.android.stats.SeriesName
+import com.avito.android.stats.StatsDSender
+import com.avito.android.stats.StatsMetric
+
+/**
+ * Send the queue time for each requested POD
+ */
+internal class StatsDKubernetesReservationListener(
+    private val statsDSender: StatsDSender,
+    private val state: KubernetesReservationState,
+    runnerPrefix: SeriesName
+) : KubernetesReservationListener {
+
+    private val queueSeriesName = runnerPrefix.append("reservation.pod.queue", multipart = true)
```

Comment:
Add please to docs 
https://avito-tech.github.io/avito-android/test_runner/Metrics/